packages/core/src/RenderingEngine/VolumeViewport3D.ts

```diff
@@ -38,6 +38,10 @@ class VolumeViewport3D extends BaseVolumeViewport {
 
   getRotation = (): number => 0;
 
+  getImageIds = (volumeId?: string): Array<string> => {
```

Comment:
the volume Viewport 3d also has imageIds. Maybe put the getImageIds insdie the baseVolumeViewport to apply for both. It is fairly similar